cli/server/convertJsonSchemas.js

```diff
@@ -237,6 +237,10 @@ const setMiscMetaProperties = (v2, meta) => {
       v2.geo_resolutions.push({key, demes: prettyDemes});
     }
   }
+  // FOOTER DESCRIPTION
```

Comment:
Since we are deliberately not making this part of v1 JSONs (i.e. meta + tree JSONs) I don't think this is necessary.